ts/components/wallet/PaymentMethodsList.tsx

```diff
@@ -88,8 +61,28 @@ const AddMethodStyle = StyleSheet.create({
 
 class PaymentMethodsList extends React.Component<Props, never> {
   public render(): React.ReactNode {
-    const { navigate } = this.props.navigation;
-
+    const paymentMethods: ReadonlyArray<IPaymentMethod> = [
+      {
+        navigateTo: this.props.navigateToAddCreditCard,
+        name: I18n.t("wallet.methods.card.name"),
+        maxFee: I18n.t("wallet.methods.card.maxFee"),
+        icon: "io-48-card"
+      },
+      {
+        navigateTo: undefined, // TODO: add route when destination is available @https://www.pivotaltracker.com/story/show/157588719
+        name: I18n.t("wallet.methods.bank.name"),
+        maxFee: I18n.t("wallet.methods.bank.maxFee"),
+        icon: "io-48-bank",
+        onPress: unavailableAlert
```

Comment:
Cant we rename `navigateTo` to onPressHandler and avoid to have a different onPress for not defined payment method?

if a payment method is implemented
onPressHandler: this.props.navigateToThePaymentMethod

if a payment method is not implemented
onPressHandler: unavailableAlert